src/OpenTelemetry.Exporter.OpenTelemetryProtocol/PersistentStorage/README.md

```diff
@@ -0,0 +1,14 @@
+# Persistent Storage APIs for OTLP Exporter
+
+The files in this folder have been copied over from
+[OpenTelemetry.PersistentStorage.Abstractions](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.PersistentStorage.Abstractions)
```

Comment:
Use permalink instead of `main` branch:

```suggestion
[OpenTelemetry.PersistentStorage.Abstractions](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/58607b7cdeb15207027a6fa4ca56e7fac897bda4/src/OpenTelemetry.PersistentStorage.Abstractions)
```